plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/PluginManager.java

```diff
@@ -129,13 +130,35 @@ public PluginManager(Config cfg) {
         httpClient = null;
     }
 
+    private String getUserAgentInformation(){
```

Comment:
Maybe it makes sense to introduce the local cache so that the information does not have to be read each time you construct a new client

Reply:
I will read a bit about local caching, don't know much about them. If you have any good pointers for me, I will appreciate them. Thanks!